nvme-print.c

```diff
@@ -362,6 +362,31 @@ const char *nvme_register_pmr_pmrszu_to_string(__u8 pmrszu)
 	}
 }
 
+void nvme_show_ctrl_register(void *bar, bool fabrics, int offset, enum nvme_print_flags flags)
+{
+	uint64_t value;
+
+	if (nvme_is_64bit_reg(offset))
+		value = mmio_read64(bar + offset);
+	else
+		value = mmio_read32(bar + offset);
+
+	if (!nvme_is_fabrics_reg(offset)) {
+		if (fabrics) {
+			printf("register: 0x%02x (%s) not fabrics\n", offset,
+			       nvme_register_to_string(offset));
+			return;
+		}
+		if (value == 0xffffffff) {
```

Comment:
Does this work for 64-bit values?

Reply:
No not work so fixed by the commit 8a750e22.